apps/juxtaposition-ui/package.json

```diff
@@ -5,7 +5,8 @@
   "license": "AGPL-3.0-only",
   "main": "./dist/server.js",
   "scripts": {
-    "dev": "tsup --watch --onSuccess \"node --enable-source-maps dist/server.js\"",
+    "dev": "nodemon -x '/bin/sh -c' -- 'tsup && node --enable-source-maps dist/server.js'",
```

Comment:
This unfortunately won't be compatible with Windows (or Mac? 🤔)

I think the best way would be to have:
`dev` - runs `nodemon --exec 'npm run dev:buildrun'`
`dev:buildrun` - uses [npm-run-all](https://www.npmjs.com/package/npm-run-all) to run two scripts sequentially using `run-s build start`